interfaces/screen-capture.idl

```diff
@@ -3,7 +3,7 @@
 // (https://github.com/tidoust/reffy-reports)
 // Source: Screen Capture (https://w3c.github.io/mediacapture-screen-share/)
 
-partial interface Navigator {
+partial interface MediaDevices {
```

Comment:
@foolip - (assuming this change has been made to the spec), this IDL change will be scraped at a later date, but is it still better to bundle it here with the changing tests to keep them behaving as expected?

Reply:
The change matches https://github.com/w3c/mediacapture-screen-share/pull/86 which hasn't been merged yet, but I think it's fine to merge spec and test update together like this. It can result in some confusion if a PR is created to revert the change due to slower propagation of the spec changes through reffy-reports, but we should just fix our tooling if it keeps happening.

Reply:
spec PR was merged, let's merge this one too.